server/index.js

```diff
@@ -9,7 +10,12 @@ app.use(bodyParser.urlencoded({extended: true}));
 app.use(morgan('tiny'));
 
 app.get('/api/movie:id', (req, res) => {
-  res.sendStatus(200);
+  var id = req.params.id.slice(1);
+  console.log(id);
+  db.getCasts(id, (casts) => {
+    res.send(casts);
```

Comment:
Here too! Sorry to be annoying with grammar haha! This can be getCast and cast. Everything else looks good to me so, this is all I can find :)

Reply:
This is great. Thanks for pointing it out. Since I've merged, I will update it in master branch directly.